apps/gp2-server/src/validation/user.validation.ts

```diff
@@ -69,6 +69,16 @@ const userPatchRequestValidationSchema: JSONSchemaType<gp2.UserPatchRequest> = {
         number: { type: 'string', nullable: true },
       },
     },
+    keywords: {
+      type: 'array',
+      items: {
+        type: 'string',
+        enum: [...keywords, '', null],
```

Comment:
can it be an empty string?

Reply:
no, and it can't be null either. This is copied over from CRN.